tests/Unit/ParallelAlgorithms/ApparentHorizonFinder/Events/Test_FindCommonHorizon.cpp

```diff
@@ -205,7 +205,8 @@ struct MockMetavariables {
 void common_horizon_event() {
```

Comment:
Again, bad commit message.  This one doesn't rename any variables at all, it adds a new variable.  The actual change is good.